src/pages/item/index.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import '../../App.css'
+import './item.css'
+import {ItemContext} from '../../context/items'
+import {BasketContext} from '../../context/basket'
+import ItemList from '../home/items-list'
+import _ from 'lodash'
+import {useParams} from 'react-router-dom'
+import ItemTemplate from '../../components/item-template/item-template'
+
+
+
+ 
+
+const Item = (props) => {
+
+  const basket = useContext(BasketContext)
+  const item = useContext(ItemContext)
+  
+  const itemPage = useContext(ItemContext)
+  const {id} = useParams()
+  const selection = _.find(ItemList, {id: itemPage.item && id} )
+  const OtherProduct = _.sampleSize(ItemList, 6)
+
+  
+  
+  const [option, setOption] = useState ('One')
+  
 
 
-function App() {
   return (
-    <div style={{ textAlign: 'center', display: 'flex', justifyContent: 'center', height: '100vh', flexDirection: 'column' }}>
-      <p>item</p>
+    <div className='Container'>
+      <div className='Nav-Bar'>
+        <button className='Round-Button' onClick={() => props.history.push('/home')}>Back To Homepage</button>
```

Comment:
I took a quick look at the styles, looks like these buttons have some minor styling issues